src/PIL/JpegImagePlugin.py

```diff
@@ -746,23 +746,28 @@ def validate_qtables(
 
     extra = info.get("extra", b"")
 
+    MAX_BYTES_IN_MARKER = 65533
     xmp = info.get("xmp")
     if xmp:
-        size = o16(31 + len(xmp))
+        overhead_len = 29
```

Comment:
Maybe add a comment that this is `len('http://ns.adobe.com/xap/1.0/'+ b"\0")`? Otherwise a reader not famliar with the matter might come to the conclusion that this is just a(nother) magic number.